### PR TITLE
Multiple matches

### PR DESCRIPTION
**Pull request by kevinkehl**
Related to https://github.com/Galooshi/emacs-import-js/issues/2

### CODE REVIEW

**Comment by kevinkehl** [comment]
cc @lencioni @trotzig maybe take a look?

File: plugin/import-js.el [hunk 1]
```diff
@@ -30,6 +30,9 @@
 
 (require 'json)
 
+(eval-when-compile
+  (require 'grizzl))
```
Comment (trotzig) [inline plugin/import-js.el]: is grizzl built in or is this something users will have to install separately?

File: plugin/import-js.el [hunk 2]
```diff
@@ -66,21 +69,51 @@
...
+                (lambda (car)
+                  (cdr (assoc 'importPath car)))
+                (cdr (assoc-string word unresolved)))))
+    (let ((file (minibuffer-with-setup-hook
+                    (lambda () (make-sparse-keymap))
+                  (grizzl-completing-read (format "Unresolved import (%s)" word)
+                                          (grizzl-make-index
+                                           paths
+                                           'files
+                                           "/Users/kevin.kehl/projects/react-board"
```
Comment (trotzig) [inline plugin/import-js.el]: This line doesn't look right.

**Comment by trotzig** [comment]
My Lisp isn't what it used to be, but overall things look good. Are you planning to start using `importjsd` as well at some point?

File: plugin/import-js.el (hunk 2 above)
Comment (kevinkehl) [inline plugin/import-js.el]: Woops left that hardcoded from my test project :)

File: plugin/import-js.el (hunk 1 above)
Comment (kevinkehl) [inline plugin/import-js.el]: Separately, but AFAIK dependencies get resolved when you use `package-install` from package.el

Comment (lencioni) [inline plugin/import-js.el]: Can you verify this?

File: plugin/import-js.el [hunk 3]
```diff
@@ -66,21 +69,51 @@
...
+                (lambda (car)
+                  (cdr (assoc 'importPath car)))
+                (cdr (assoc-string word unresolved)))))
+    (let ((file (minibuffer-with-setup-hook
+                    (lambda () (make-sparse-keymap))
+                  (grizzl-completing-read (format "Unresolved import (%s)" word)
+                                          (grizzl-make-index
+                                           paths
+                                           'files
+                                           import-js-current-project-root
```
Comment (kevinkehl) [inline plugin/import-js.el]: I've fixed this to reference the current project root (set above).

**Comment by kevinkehl** [comment]
@trotzig @lencioni I'd like to merge this so I can verify with a clean install. I'll monitor the MELPA deploy and fix/revert if it doesn't work as expected.

MELPA stable should not be effected, for anyone using the stable version (as they probably should be, normally).

**Comment by trotzig** [comment]
Merge at will! (you have the right privileges in this project, right?)

**Comment by kevinkehl** [comment]
Yep! But as it's a larger change, just wanted eyes on =D

**Pull request merged**